testCourse/courseInstances/public/infoCourseInstance.json

```diff
@@ -3,6 +3,9 @@
   "longName": "Public",
   "shareSourcePublicly": true,
   "allowAccess": [
-    { "institution": "Any", "startDate": "1800-01-19T00:00:01", "endDate": "2400-05-13T23:59:59" }
```

Comment:
Why was `"institution": "Any"` dropped?

Reply:
It's going to be dropped in the follow-up PR, it doesn't really matter